README.md

```diff
@@ -27,6 +27,17 @@ This will create the required namespace, custom resource definition, roles, depl
 Note: The above link is set to use the configuration for the latest version of the Update Operator, `v0.2.1`.
 Be sure to use the git tag for the Update Operator release that you plan to deploy.
 
+#### Features
+
+##### exclude from load balancer before draining
+`EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC` can be used to enable the feature to exclude the node from load balancer before draining.
+When `EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC` is 0 (default), the feature is disabled.
+When `EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC` is set to a positive integer, bottlerocket update operater will exclude the node from 
+load balancer, wait `EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC` seconds before draining the pods on the node.
+
+To enable this feature, go to `.env`, change `EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC` to a positive integer value,
+run `cargo build` to regenerate `bottlerocket-update-operator.yaml` and follow the normal installation guideline.
```

Comment:
I think instead of directing folks to rebuild the project, we should have folks modify the `.yaml` document describing the pod deployment to modify the environment variable.

When we begin delivering this as a Helm chart, we can make that variable configurable via Helm.

Reply:
Updated readme using .yaml for user. I would like to have a separate PR to update the development part.